### PR TITLE
High Deposit/Withdrawals

### PR DESCRIPTION
**Pull request by e1Ru1o**

### CODE REVIEW

File: OlympusDAO-Agent-Suit/DepositsAndWithdrawals/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+## Supported Chains
+
+- Ethereum
+
+## Alerts
+
+- olympus-treasury-5-1
+  - Fired when a high deposit ocur
+  - Severity is always set to "info" (mention any conditions where it could be something else)
+  - Type is always set to "suspicious" (mention any conditions where it could be something else)
```
Comment (mauricio1802) [inline OlympusDAO-Agent-Suit/DepositsAndWithdrawals/README.md:17]: Remove comments between `()`.

File: OlympusDAO-Agent-Suit/DepositsAndWithdrawals/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  - Metadata contains:
+    - `token`: Address of the deposited token
+    - `amount`: Amount of tokens deposited
+    - `value`: Value in OHM
+
+- olympus-treasury-5-2
+  - Fired when a high withdrawal ocur
+  - Severity is always set to "info" (mention any conditions where it could be something else)
+  - Type is always set to "suspicious" (mention any conditions where it could be something else)
```
Comment (mauricio1802) [inline OlympusDAO-Agent-Suit/DepositsAndWithdrawals/README.md:26]: Remove comments between ().

**Review by mauricio1802 — CHANGES_REQUESTED** [review]
Change PR destination to `@olympusDAO`

File: OlympusDAO-Agent-Suit/DepositsAndWithdrawals/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+> Contract: 0x31f8cc382c9898b273eff4e0b7626a6987c846e8
+
+## Supported Chains
+
+- Ethereum
+
+## Alerts
+
+- olympus-treasury-5-1
+  - Fired when a high deposit ocur
```
Comment (CeliktepeMurat) [inline OlympusDAO-Agent-Suit/DepositsAndWithdrawals/README.md:15]: typo -> "occurs"

File: OlympusDAO-Agent-Suit/DepositsAndWithdrawals/README.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  - Severity is always set to "info" 
+  - Type is always set to "suspicious" 
+  - Metadata contains:
+    - `token`: Address of the deposited token
+    - `amount`: Amount of tokens deposited
+    - `value`: Value in OHM
+
+- olympus-treasury-5-2
+  - Fired when a high withdrawal ocur
```
Comment (CeliktepeMurat) [inline OlympusDAO-Agent-Suit/DepositsAndWithdrawals/README.md:24]: typo -> "occurs"

**Review by CeliktepeMurat — CHANGES_REQUESTED** [review]
Change destination of PR from `main`  to `@olympusDAO`

**Review by CeliktepeMurat — APPROVED** [review]

**Review by mauricio1802 — APPROVED** [review]

**Pull request merged**